src/odemis/driver/pigcs.py

```diff
@@ -2843,8 +2840,8 @@ def __init__(self, name, role, port, axes, baudrate=38400,
         controllers = {} # address -> kwargs (axes, dist_to_steps, min_dist, vpms...)
         for axis, (add, channel, isCL) in axes.items():
             if isinstance(channel, int):
-                channel = str(channel)
-            channel = channel.encode("ascii")  # bytes
+                channel = "%d" % channel
+            channel = channel  # str
```

Comment:
That's a little redundant code. Just move the `# str` comment to the line above, and drop this line.